lib/logstash/inputs/jdbc.rb

```diff
@@ -160,6 +160,14 @@ class LogStash::Inputs::Jdbc < LogStash::Inputs::Base
   # Whether to force the lowercasing of identifier fields
   config :lowercase_column_names, :validate => :boolean, :default => true
 
+  # Default encoding applied to all columns unless no specific encoding is defined
+  # in the columns_to_encode variable. By default this value is not defined
+  config :default_encoding, :validate => :string
```

Comment:
I suggest we keep the same option naming as in the codecs and use `:charset` instead of `default_encoding`.